spec/GraphQLOverHTTP.md

```diff
@@ -401,8 +401,8 @@ And the body:
 
 # Response
 
-When a server receives a _GraphQL-over-HTTP request_, it must return a
-well‐formed response. The server's response describes the result of validating
+When a server receives a well-formed _GraphQL-over-HTTP request_, it must return a
+well‐formed [GraphQL response](#sec-Response). The server's response describes the result of validating
```

Comment:
I think this is supposed to link to https://spec.graphql.org/October2021/#sec-Response - however that does not seem to work correctly in the rendered output:

![image](https://user-images.githubusercontent.com/12158000/195295424-2df9caab-0e84-43f9-ad0f-711763b4ed82.png)

How can we correct this?

Reply:
By italicising we can leverage spec-md's referencing feature to reference the spec:

```suggestion
well‐formed _GraphQL response_. The server's response describes the result of validating
```

Enabled by:

https://github.com/graphql/graphql-over-http/blob/466f5db37eac577fe999d9e8118317919b86385a/spec/metadata.json#L7

Reply:
Thank you, I corrected the remaining references.